packages/cli/templates/default-plugin/package.json.hbs

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "@backstage/plugin-{{id}}",
+  "name": "{{#if scopeName}}{{scopeName}}/{{/if}}/plugin-{{id}}",
```

Comment:
probably just template the entire thing as `{{ packageName }}` tbh, and put this together in the calling code.